items:
  prob_catastrophic_failure: 0.2
  cheezit_big_original:
    supported: true
    bonus: 0
    force: 100 
  genuine_joe_plastic_stir_sticks:
    supported: true
    bonus: 0
    force: 15
  crayola_64_ct:
    supported: true
    bonus: 0
    force: 50
  expo_dry_erase_board_eraser:
    supported: true
    bonus: 0
    force: 1
  elmers_washable_no_run_school_glue:
    supported: true
    bonus: 0
    force: 125
  kyjen_squeakin_eggs_plush_puppies:
    supported: true
    bonus: 1
    force: 255
  oreo_mega_stuf:
    supported: true
    bonus: 0
    force: 50
  champion_copper_plus_spark_plug:
    supported: true
    bonus: 0
    force: 15
  munchkin_white_hot_duck_bath_toy:
    supported: true
    bonus: 1
    force: 50
  mommys_helper_outlet_plugs:
    supported: true
    bonus: 0
    force: 1
  sharpie_accent_tank_style_highlighters:
    supported: true
    bonus: 0
    force: 50
  stanley_66_052:
    supported: true
    bonus: 3
    force: 15
  safety_works_safety_glasses:
    supported: true
    bonus: 1
    force: 40
  paper_mate_12_count_mirado_black_warrior:
    supported: true
    bonus: 0
    force: 50
  mark_twain_huckleberry_finn:
    supported: true
    bonus: 3
    force: 1
  first_years_take_and_toss_straw_cup:
    supported: true
    bonus: 0
    force: 50
  feline_greenies_dental_treats:
    supported: true
    bonus: 0
    force: 255
  laugh_out_loud_joke_book:
    supported: true
    bonus: 3
    force: 1
  kong_air_dog_squeakair_tennis_ball:
    supported: true
    bonus: 1
    force: 255
  highland_6539_self_stick_notes:
    supported: true
    bonus: 0
    force: 15
  kong_sitting_frog_dog_toy:
    supported: true
    bonus: 1
    force: 255
  kong_duck_dog_toy:
    supported: true
    bonus: 1
    force: 255
  rolodex_jumbo_pencil_cup:
    supported: true
    bonus: 2
    force: 50
  mead_index_cards:
    supported: true
    bonus: 0
    force: 50
  dr_browns_bottle_brush:
    supported: true
    bonus: 2
    force: 1
item_priorities:
  - crayola_64_ct
  - expo_dry_erase_board_eraser
  - elmers_washable_no_run_school_glue
  - cheezit_big_original
  - kyjen_squeakin_eggs_plush_puppies
  - genuine_joe_plastic_stir_sticks
  - munchkin_white_hot_duck_bath_toy
  - mommys_helper_outlet_plugs
  - sharpie_accent_tank_style_highlighters
  - stanley_66_052
  - safety_works_safety_glasses
  - paper_mate_12_count_mirado_black_warrior
  - mark_twain_huckleberry_finn
  - first_years_take_and_toss_straw_cup
  - feline_greenies_dental_treats
  - laugh_out_loud_joke_book
  - kong_air_dog_squeakair_tennis_ball
  - mead_index_cards
  - kong_sitting_frog_dog_toy
  - kong_duck_dog_toy
  - highland_6539_self_stick_notes
  - oreo_mega_stuf
  - dr_browns_bottle_brush
  - champion_copper_plus_spark_plug
  - rolodex_jumbo_pencil_cup
